.gitlab-ci.yml

```diff
@@ -62,3 +62,29 @@ build_wheel:
     # create the wheel and upload to pypi
     - sh scripts/create_assembl_wheel.sh
     - twine upload -u bluenove --skip-existing dist/*
+
+deploy_to_dev:
+  stage: deploy_to_dev
+  only:
+    - develop
```

Comment:
it will be `tmp_master_for_ci_cd` and then master here

Reply:
makes sense